04.ls/ls.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+FIND_FILES = Dir.glob('*')
+SPLIT_NUMBER = 3
+
+def splitting_array
+  max_length = FIND_FILES.length / SPLIT_NUMBER
+  FIND_FILES.length % SPLIT_NUMBER != 0 ? max_length += 1 : max_length
+
+  split_array = FIND_FILES.each_slice(max_length).to_a
+  split_array[2] += Array.new(split_array[0].length - split_array[2].length, nil) if FIND_FILES.length % SPLIT_NUMBER != 0
```

Comment:
```suggestion
  hoge = split_array[0].length - split_array[2].length
  split_array[2] += Array.new(hoge, nil) if FIND_FILES.length % SPLIT_NUMBER != 0
```

ちょっと横に長いので一度変数に入れた方が読みやすいと思います。
https://twitter.com/jnchito/status/1326468865103073280

あと、このままだと3列から5列に変更してくださいと言われたときに修正が必要になるので、列番号の決め打ちを避けたいなーと思いました。